edx_sga/sga.py

```diff
@@ -534,22 +643,32 @@ def _file_storage_path(self, sha1, filename):
         return path
 
 
-def _get_sha1(file):
-    BLOCK_SIZE = 2**10 * 8  # 8kb
+def _get_sha1(_file):
+    """
+    Get file sha1
+    :param _file:
+    """
+    block_size = 2**10 * 8  # 8kb
```

Comment:
Being consistent is very important. If you used 1<<10 at one place, no need to use 2**10 at the other place to achieve the same value. Moreover, both 1<<10 and 2**10 are not any "clearer" or "more accurate than simple "1024".